### DISCUSSION TITLE
源码中有一处两个零宽度空格

### DISCUSSION THREAD

**Discussion by lostpg** [the1812/Bilibili-Evolved#3391] [综合讨论 / General]
https://github.com/the1812/Bilibili-Evolved/blob/407754c3253f8d7fbbdac8cfd5499e1df6156c10/registry/lib/components/video/player/common/speed.ts#L119-L121

位于第一个 `component` 的 `c` 前有两个 U+200B ZERO WIDTH SPACE，如果是无意加入的话就麻烦删除一下。

**Comment by the1812** [comment]
可以的

**Reply by lostpg** [reply]
Cool
